src/Commands/MpxImporter.php

```diff
@@ -98,36 +96,39 @@ public function __construct(EntityTypeManagerInterface $entityTypeManager, DataO
    * @param string $media_type_id
    *   The media type ID to import for.
    *
-   * @usage media_mpx-import mpx_video
+   * @option batch_size An integer with the number of items to import per batch.
+   *
+   * @usage media_mpx:import {bundle}
    *   Import all mpx data for the mpx_video media type.
    *
    * @command media_mpx:import
    * @aliases mpxi
    */
-  public function import(string $media_type_id) {
-    $media_type = $this->loadMediaType($media_type_id);
-
-    // Only return the media ids to add to the queue.
-    $field_query = new Fields();
-    $field_query->addField('id');
-    $query = new ObjectListQuery();
-    $query->add($field_query);
-
-    $results = $this->select($media_type, $query);
+  public function import(string $media_type_id, $options = ['batch_size' => 100]) {
+    $queue_service = \Drupal::getContainer()->get('media_mpx.service.queue_video_imports');
```

Comment:
This class is already using DI, so use it here too?

Reply:
@deviantintegral great catch. In fact cleaning this up I also noticed 2 other dependencies are not even needed anymore as part of the refactor, and removed those as well \o/. This is much thinner now 🎉